### ISSUE TITLE
Select2 508 issues

### ISSUE THREAD

**Issue by prerakdalal** [select2/select2#4303]
Hi,

I have been using this plugin and it has some great online documentation as well as functionalities.
One of the issues I have been facing is, It doesn't seem to be 508 compliant. The screen reader doesn't read the suggestions and also when multiselect is enabled it doesn't read my selections. 

I am using NVDA screen reader for my testing. I am planning to use this plugin in a federal project and it requires the 508 compliance. Any quick help on this will be greatly appreciated.

**Comment by kevin-brown** [comment]
This is effectively a duplicate of https://github.com/select2/select2/issues/3744. As such I'm going to merge them together.

I personally have not been trained to work with any of the generally available screen readers, which makes fixing any accessibility issues _very_ difficult from my side of things. I would **love to** support screen readers fully and without any issues, but I honestly have _very few_ starting places. Once we get the support in initially, it'll actually be considerably easier to ensure that the support remains. We just need to get the initial support for it.

Any recommendations, tips, or tricks that might be helpful for us when adding support would be _greatly appreciated_ in #3744.

**Comment by prerakdalal** [comment]
Thanks kevin,  I will follow the thread in #3744
